tree/tree/inc/TLeafI.h

```diff
@@ -49,6 +50,8 @@ class TLeafI : public TLeaf {
    virtual void    PrintValue(Int_t i=0) const;
    virtual void    ReadBasket(TBuffer &b);
    virtual void    ReadBasketExport(TBuffer &b, TClonesArray *list, Int_t n);
+   virtual bool    ReadBasketFast(TBuffer&, Long64_t);
+   virtual bool    ReadBasketSerialized(TBuffer&, Long64_t) {return GetDeserializeType() == kInPlace; }
```

Comment:
The semantic of this routine are unclear (i.e. need at least a doc string).   Why are the arguments never used?